include/crm/crm.h

```diff
@@ -51,7 +51,7 @@ extern "C" {
  * >=3.0.13: Fail counts include operation name and interval
  * >=3.2.0:  DC supports PCMK_LRM_OP_INVALID and PCMK_LRM_OP_NOT_CONNECTED
  */
-#  define CRM_FEATURE_SET		"3.4.1"
+#  define CRM_FEATURE_SET		"3.4.2"
```

Comment:
This has to be 3.5.0 to have any effect on the cluster (i.e. ensuring a newer node can't be DC until all nodes have been upgraded). The minor-minor is just for external code to be able to test for new features.